report_qweb_encrypt/models/ir_actions_report.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 Creu Blanca
+# Copyright 2020 Ecosoft Co., Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import time
+import logging
+from odoo import fields, models, _
+from io import BytesIO
+from odoo.tools.safe_eval import safe_eval
+from odoo.exceptions import ValidationError
+
+
+_logger = logging.getLogger(__name__)
+try:
+    from PyPDF2 import PdfFileReader, PdfFileWriter
+except ImportError as err:
+    _logger.debug(err)
+
+
+class IrActionsReport(models.Model):
+
+    _inherit = 'ir.actions.report'
+
+    encrypt = fields.Boolean(
+        string="Encryption",
+    )
+    encrypt_password_option = fields.Selection(
```

Comment:
With this is not required the flag. Encrypt flag is true if this value is set. WDYT?

Reply:
Agreed, I think we can remove encrypt_password_option and use encrypt as the options itself.

Reply:
WDYT?

![image](https://user-images.githubusercontent.com/1973598/103150400-6cc62400-47a6-11eb-966c-fdff6b84dd60.png)

Reply:
Gr8